project/gen2.scala

```diff
@@ -168,8 +168,8 @@ object gen2 {
       },
 
       enum("Instrument") {
-        type InstrumentRec = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'tccValue -> String, 'obsolete -> Boolean`.T
-        val io = sql"select id, id tag, short_name, long_name, tcc_value, obsolete from e_instrument".query[(String, InstrumentRec)].list
+        type InstrumentRec = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'obsolete -> Boolean`.T
+        val io = sql"select id, id tag, short_name, long_name, obsolete from e_instrument".query[(String, InstrumentRec)].list
         io.transact(xa).unsafePerformIO
```

Comment:
This is actually part of the build.  Each one of these is generating an enum data type from information in the database so it needs to do some IO.